src/main/java/org/junitpioneer/internal/PioneerAnnotationUtils.java

```diff
@@ -10,6 +10,8 @@
 
 package org.junitpioneer.internal;
 
+import static java.util.stream.Collectors.toUnmodifiableList;
```

Comment:
Given your [recent tweet](https://twitter.com/nipafx/status/1613812216419028993), and the changes in this commit, I assume you prefer to statically import methods from `Collectors`?! 😉 Then, we should probably enforce this via Spotless.

Reply:
I'm not sure we need to enforce such details, but we _can_. What would that rule look like?

Reply:
I'm guessing [OpenRewrite](https://github.com/openrewrite/rewrite) or [Refaster](https://errorprone.info/docs/refaster) could be used to auto-format code to use this style.

Reply:
@jbduncan we already have Checkstyle and Spotless. I couldn't find a corresponding functionality in Spotless (which I prefer, because it not just detects violations, but also fixes them), but Checkstyle can do this this:

https://stackoverflow.com/a/47649243

So I guess we could extend our [`checkstyle.xml`](https://github.com/junit-pioneer/junit-pioneer/blob/main/.infra/checkstyle/checkstyle.xml) as follows:

```xml
<module name="RegexpSinglelineJava">
	<property name="format" value="import java.util.stream.Collectors;"/>
	<property name="message" value="Collectors must be statically imported"/>
</module>

Reply:
@beatngu13 Okay, if there's no desire to include another tool, then I totally understand!

But for reference, here is OpenRewrite's doc on auto-fixing Checkstyle issues: https://docs.openrewrite.org/running-recipes/popular-recipe-guides/automatically-fix-checkstyle-violations

Reply:
A potential con for OpenRewrite: In my experience, its Gradle plugin runs a fair bit slower than Spotless, but I only have experience with it on a trivially small project, so YMMV.

Reply:
@jbduncan oh, nice – didn't know that, thank you!

Reply:
I have updated the checkstyle ruleset but I'm not sure about this approach - we _could_ use the [IllegalImport](https://checkstyle.sourceforge.io/config_imports.html#IllegalImport) check.

Reply:
I'm fine with both, but `IllegalImport` seems more specific, so feel free to use it.

Reply:
I like the `IllegalImport` rule too as it's to covered the `sun.` packages

Reply:
I think I actually prefer this rule after thinking about it. `IllegalImport` is specifically about _not using_ certain classes/packages. This is not the case here - we _want to_ use `Collectors` but only with static imports.